CHANGELOG.md

```diff
@@ -91,7 +93,8 @@ error types when using tools like `probe-rs` for logging over debuggers.
 
 Initial release
 
-[Unreleased]: https://github.com/rust-embedded-community/serde-json-core/compare/v0.5.1...HEAD
+[Unreleased]: https://github.com/rust-embedded-community/serde-json-core/compare/v0.6.0...HEAD
+[v0.5.1]: https://github.com/rust-embedded-community/serde-json-core/compare/v0.5.1...v0.6.0
```

Comment:
Wrong version

Reply:
Fixed in abd58786df88cc4a612178a0ddce29f00c976424